client/post-editor/editor-title/index.jsx

```diff
@@ -118,17 +97,17 @@ const EditorTitle = React.createClass( {
 						isEditable={ isPermalinkEditable } />
 				}
 				<TrackInputChanges onNewValue={ this.recordChangeStats }>
-					<FormTextInput
+					<TextareaAutosize
 						{ ...omit( this.props, Object.keys( this.constructor.propTypes ) ) }
 						className="editor-title__input"
 						placeholder={ this.translate( 'Title' ) }
 						onChange={ this.onChange }
 						onBlur={ this.onBlur }
-						onFocus={ this.onFocus }
 						autoFocus={ isNew && ! isMobile() }
 						value={ post ? post.title : '' }
 						aria-label={ this.translate( 'Edit title' ) }
-						ref="titleInput" />
+						ref="titleInput"
+						rows="1" />
```

Comment:
>Right, I mean the default initial value for the component.

Still not certain I'm following, but an empty field will default to `1` rows, and grow as necessary via `autosize`. Or are you suggesting we use `rows="1"` as a default prop for `<TextareaAutosize />` ?